src/Clockify/ClockifySetupDialog.cs

```diff
@@ -77,12 +79,18 @@ public class ClockifySetupDialog : ComponentDialog
                 var userProfile = await _userState.CreateProperty<UserProfile>("UserProfile")
                     .GetAsync(promptContext.Context, () => new UserProfile(), cancellationToken);
 
-                string? userId = _clockifyService.GetCurrentUserAsync(token).Result.Id;
+                CurrentUserDto currentUser = await _clockifyService.GetCurrentUserAsync(token);
```

Comment:
When using await, the exception raised is flattened, so we should not catch an AggregateException